codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectDecoder.java

```diff
@@ -1127,4 +1149,8 @@ public boolean process(byte value) {
             return ISO_CONTROL_OR_WHITESPACE[128 + value];
         }
     };
+
+    private static boolean isControlOrWhitespaceAsciiChar(byte b) {
+        return ISO_CONTROL_OR_WHITESPACE[128 + b];
```

Comment:
Do we need to ensure this will not throw an IndexOutOfBoundsException ?

Reply:
It cannot, in theory, the array should be big enough to cover the full spectrum of byte values (but let me check)